internal/deep/dpobjects/support.go

```diff
@@ -0,0 +1,54 @@
+package dpobjects
+
+import (
+	"github.com/amp-labs/connectors/common/handy"
+	"github.com/amp-labs/connectors/internal/deep/requirements"
+)
+
+// Registry lists object names allowed to perform operation.
+// By default, everybody can pass.
+type Registry struct {
```

Comment:
This naming is not intuitive. 

We can make it SupportedObjectMapRegistry or something to make it very clear so that everyone can understand what this is about.

Reply:
Yeah, this one was hard to name as it also implements the interface that should be named somehow too.
I removed the occurence of object since the package name has this.
`dpobjects.Registry` - but it doesn't reflect that it is supported... Maybe `dpobjects.SupportRegistry`.